readme.md

```diff
@@ -0,0 +1,13 @@
+eBPF
```

Comment:
Could we add a huge banner saying "Work in progress. The API will definitely change"?

Reply:
Yeah, I'll do that in a PR straight after this if that is OK?

Reply:
Sure.